### PR TITLE
Sort search results by content type by default

### LINKED ISSUE #233: Research how to show Collections over items regardless of relevance
iDC-7, iDC-30

iDC-7: Normal Path: Step 9b

### PR DESCRIPTION
**Pull request by jabrah**
resolves https://github.com/jhu-idc/iDC-general/issues/233

This configuration change force search results to be sorted by type (collection_object, islandora_object) first, before relevance, effectively grouping search results by type. If a user is to select a "Sort By" option in the UI, the option will override this behavior. For example, if the user sorts by "title" in the UI, the results will no longer by sorted by type, only by title.

This needs a review to make sure it meets our use case requirement for sorting

### CODE REVIEW

File: codebase/config/sync/views.view.solr_search_content.yml [hunk 1]
```diff
@@ -20,11 +19,7 @@ dependencies:
     - field.storage.node.field_date_created
     - field.storage.node.field_date_published
     - field.storage.node.field_description
-    - field.storage.node.field_digital_identifier
```
Comment (bseeger) [inline codebase/config/sync/views.view.solr_search_content.yml:23]: what does it mean to remove this? What effect does it have on things?

Comment (jabrah) [reply codebase/config/sync/views.view.solr_search_content.yml:23]: The only change I made was to add content type sorting through the admin UI, then ran `make config-export`. I don't know why Drupal changed the rest of the config.

**Review by bseeger — APPROVED** [review]
We looked at the configuration changes and determined they are innocuous. 

These changes look good.

**Comment by bseeger** [comment]
After playing with the development branch, I realized that the sort order is wrong for content type. The repository items are showing up first before the collections and it needs to be the other way around.  The configuration for that sort field should be ASC and not DESC.  Given that @jabrah  is out this week, I'll make a quick PR to adjust this.

**Pull request merged**